### ISSUE TITLE
Multiple Repository Management

### ISSUE THREAD

**Issue by ghost** [vgmoose/wiiu-hbas#21]
Perhaps add a repo selection gui to the app itself?

**Comment by vgmoose** [comment]
Yep, this would be a nice addition, instead of the hacky repository.txt setup.

Even nicer would be the ability to add/remove repos directly within the app, but as that would require a keyboard, resolving this issue would be a good first step.

**Comment by tomsterley** [comment]
Why not use one of the wii u keyboard libs developed?

**Comment by ghost** [comment]
In fact, Im gonna fork this and see if I can get that implemented, if I do, i will make a pull request

EDIT: DevkitPro hates me, so im ditching that idea :|

**Comment by vgmoose** [comment]
It may be possible to use the real Wii U keyboard though when compiling to RPX using [WUT](https://github.com/decaf-emu/wut). The reason I haven't integrated the Wii U keyboard homebrew is it's designed for OSScreen.

If you want to make a GX2 keyboard library though, I I can think of a few homebrews which would be able to use it.

Opening this back up cause it's still an issue! Also going along with a repo selector, would be the redesigning of "repository.txt"

Probably just a list of repository urls, each one on a new line, will work fine.

**Comment by tomsterley** [comment]
Or you could list like "WiiUBru Repo", "VGMoose Repo", etc.

**Comment by ghost** [comment]
Yeah, even better is you could do (url)[title] in repository.txt, so it would have a menu within hbas like: [title] [title], etc.
